pos_analytic_by_config/i18n/de.po

```diff
@@ -0,0 +1,53 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# * pos_analytic_by_config
+#
+# Translators:
+# Rudolf Schnapka <rs@techno-flex.de>, 2017
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 8.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-01-20 23:05+0000\n"
+"PO-Revision-Date: 2017-01-20 23:05+0000\n"
+"Last-Translator: Rudolf Schnapka <rs@techno-flex.de>, 2017\n"
+"Language-Team: German (https://www.transifex.com/oca/teams/23907/de/)\n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#. module: pos_analytic_by_config
+#: model:ir.model.fields,field_description:pos_analytic_by_config.field_pos_config__account_analytic_id
+#: model_terms:ir.ui.view,arch_db:pos_analytic_by_config.view_pos_config_form
+msgid "Analytic Account"
+msgstr "Kostenstelle"
+
+#. module: pos_analytic_by_config
+#: model_terms:ir.ui.view,arch_db:pos_analytic_by_config.view_pos_config_form
+msgid "Analytic Account for this config."
+msgstr ""
+
+#. module: pos_analytic_by_config
+#: model_terms:ir.ui.view,arch_db:pos_analytic_by_config.view_pos_config_form
+msgid "Analytic Account that these orders link to"
+msgstr ""
+
+#. module: pos_analytic_by_config
+#: model:ir.model,name:pos_analytic_by_config.model_account_invoice_line
+msgid "Invoice Line"
+msgstr ""
+
+#. module: pos_analytic_by_config
+#: model:ir.model,name:pos_analytic_by_config.model_pos_config
+msgid "Point of Sale Configuration"
+msgstr ""
+
+#. module: pos_analytic_by_config
+#: model:ir.model,name:pos_analytic_by_config.model_pos_order
+msgid "Point of Sale Orders"
+msgstr ""
+
+#~ msgid "Point of Sale"
+#~ msgstr "Verkaufsstelle"
```

Comment:
```suggestion
```

Reply:
I don't understand this suggestion :thinking:

Reply:
My suggestion is related to translations purge.

Reply:
Isn't that something the bot does automatically?

Reply:
I m not really sure, IMO not but it's not really important to block merge.

Reply:
I think they are not purged, only carried all time.

Reply:
Ok, let's purge them :)